horace_core/admin/cache.py

```diff
@@ -162,10 +157,11 @@ def process_file(argi):
 if __name__ == '__main__':
     """Read file using multiple threads to place file into CEPH cache"""
 
-    parser = argparse.ArgumentParser(add_help=True, version='0.1',description='Read file using multiple threads to place file into CEPH cache')
-    parser.add_argument('filename',action='store', type=str,default="",help='file to read to cache')
-    parser.add_argument('-n',action='store', dest='nthreads',type=int,default=16,help='number of threads to process file. Default is 16 threads')
-    parser.add_argument('-b',action='store', dest='buffer',type=int,default=4096,help='Buffer size to read each chunk of data. Default is 4096 bytes.')
+    parser = argparse.ArgumentParser(add_help=True,description='Read file using multiple threads to place file into CEPH cache')
+    parser.add_argument('filename',action='store', type=str,default="",help='file to read for caching in CEPHS')
+    parser.add_argument('--nthreads','-n',action='store', dest='nthreads',type=int,default=16,help='number of threads to process file. Default is 16 threads')
+    parser.add_argument('--buffer','-b',action='store', dest='buffer',type=int,default=4096,help='Buffer size to read each chunk of data. Default is 4096 bytes.')
+    parser.add_argument('-version',action = 'version',version = '0.1')
```

Comment:
Standards suggest using `--` for long names

Reply:
that's what is done